lib/posix/spawn/child.rb

```diff
@@ -148,8 +154,9 @@ def exec!
       rescue Object => boom
         [stdin, stdout, stderr].each { |fd| fd.close rescue nil }
         if @status.nil?
-          ::Process.kill('TERM', pid) rescue nil
-          @status = waitpid(pid)      rescue nil
+          kill_pid = @pgroup_kill ? pid * -1 : pid
```

Comment:
> Also, fwiw, the docs for Process.kill mention negating the signal, not the pid, to kill a group. Not sure which is more idiomatic.

Weird, seems to work w/ either a negative pid or "-TERM" signal string. I'll go with the documented approach here.